pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -2079,6 +2080,87 @@ func TestListUsersCycleDetection(t *testing.T) {
 	})
 }
 
+func TestListUsersDepthExceeded(t *testing.T) {
+	model := `model
+	schema 1.1
+		type user
+
+	type folder
+		relations
+			define parent: [folder]
+			define viewer: [user] or viewer from parent`
+
+	depthNotExceededTuples := []*openfgav1.TupleKey{
```

Comment:
Nit: this is an odd variable name since you use these for both tests. Tuples by themselves don't exceed any limits

Reply:
It makes sense to me 🤷‍♂️ . Literally speaking, you're correct but the particular collection of tuples is a necessary condition to trigger the depth exceeded error. Really don't know what a better alternative would be. Can you suggest one?

Reply:
Just `tuples` 😄